src/Raven.Server/Documents/Sharding/ShardedDocumentDatabase.cs

```diff
@@ -64,6 +66,11 @@ protected override ReplicationLoader CreateReplicationLoader()
         return new ShardReplicationLoader(this, ServerStore);
     }
 
+    protected override ShardSubscriptionStorage CreateSubscriptionStorage(ServerStore serverStore)
+    {
+        return new ShardSubscriptionStorage(this, serverStore);
```

Comment:
you probably do not need DocumentDatabase.InitializeSubscriptionStorage method. You can pass the appropriate name via ctor now here?

Reply:
done,